testsuite/gnat2goto/tests/arrays_assign_nonoverlap/arrays_assign_nonoverlap.adb

```diff
@@ -0,0 +1,9 @@
+procedure Arrays_Assign_Nonoverlap is
+   type Arr_Long is array (1..4) of Integer;
+   Full_Arr : Arr_Long := (1,2,3,4);
+   Part1_Arr : Arr_Long := (1,2);
```

Comment:
??? This isn't actually valid Ada though? `Arr_Long` is a size 4 array, you can't initialise it with a size 2 aggregate (it's syntactically valid but would raise an exception at runtime if we supported exceptions).

Reply:
Ok, fixed.